projects/BEVFusion/configs/bevfusion_voxel0075_second_secfpn_8xb4-cyclic-20e_nus-3d.py

```diff
@@ -246,12 +248,16 @@
         rot_lim=[-5.4, 5.4],
         rand_flip=True,
         is_train=True),
+    # dict(
+    #     type='GlobalRotScaleTrans',
+    #     resize_lim=[0.9, 1.1],
+    #     rot_lim=[-0.78539816, 0.78539816],
+    #     trans_lim=0.5,
+    #     is_train=True),
     dict(
         type='GlobalRotScaleTrans',
-        resize_lim=[0.9, 1.1],
-        rot_lim=[-0.78539816, 0.78539816],
-        trans_lim=0.5,
-        is_train=True),
+        rot_range=[-0.78539816, 0.78539816],
+        scale_ratio_range=[0.9, 1.1]),
     dict(type='RandomFlip3D'),
```

Comment:
I noticed that the implementation of RandomFlip3D in mmdet3d and BEVFusion is different. You can check it.